Bots/DotNet/Skills/CodeFirst/WaterfallSkillBot/Bots/SkillBot.cs

```diff
@@ -44,6 +44,11 @@ public override async Task OnTurnAsync(ITurnContext turnContext, CancellationTok
             await _conversationState.SaveChangesAsync(turnContext, false, cancellationToken);
         }
 
+        protected override Task<InvokeResponse> OnInvokeActivityAsync(ITurnContext<IInvokeActivity> turnContext, CancellationToken cancellationToken)
```

Comment:
What does this do?

Reply:
I was testing something and forgot to remove this.